microservices/password-manager/controllers/primistore/primistore-controller.ts

```diff
@@ -35,16 +37,16 @@ const storage = multer.diskStorage({
 });
 const upload = multer({ storage: storage });
 
-const passwordCreationHandler = async (req, res, logger) => {
+const passwordCreationHandler = async (req: Request, res: Response, logger: winston.Logger) => {
   const password_uid = req.body.identifier;
 
   const { key, iv } = generateAESKeyIV();
   if (
-    key.type === CommandOutputType.Error ||
-    iv.type === CommandOutputType.Error
+    key.type === "error" ||
```

Comment:
The type is an enum, and will return 0 or 1 and not strings.

Reply:
Same for all other command output type comparisons.

Reply:
CommandOutputType is already exposed by the latest version of command-executor-lib, do not compare with a string.

Reply:
I changed the type into string, and typescript autocompletion suggests `success`, `error`, and I believe there is no reason to use enums!